### PR TITLE
Refactor/esl a11ty group

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by Natalie-Smirnova — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-a11y-group/core/esl-a11y-group.ts [hunk 1]
```diff
@@ -35,37 +42,19 @@ export class ESLA11yGroup extends ESLBaseElement {
...
   protected _onKeydown(e: KeyboardEvent): void {
     const target = e.target as HTMLElement;
-
     if (!this.$targets.includes(target)) return;
 
-    if ([ARROW_UP, ARROW_LEFT].includes(e.key)) {
-      this.goTo('prev', target);
-      e.preventDefault();
-    }
+    const groupTarget = (this.constructor as typeof ESLA11yGroup).KEY_MAP[e.key];
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-a11y-group/core/esl-a11y-group.ts:53]: Yep, as @dshovchko mentioned we can use it with the inheritance mechanism in that case.
JS inheritance is really cool thing in terms of the ability to selectively change some props or methods in the inheritor, including static members of the base class. 
```ts
  class ClassA {
      static member = [1, 2];
      
      fixedPrint() {
         console.log(ClassA,member);
      }
      print() {
         console.log(this.constructor as typeof ClassA),member);
      }
  }
  class ClassB extends ClassA {
      static member = [3, 4];
  }
  
  (new A()).fixedPrint() // 1, 2
  (new B()).fixedPrint() // 1, 2 
  (new A()).print() // 1, 2
  (new B()).print() // 3, 4 
```

**Review by abarmina — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

File: src/modules/esl-a11y-group/core/esl-a11y-group.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nattallius) [reply src/modules/esl-a11y-group/core/esl-a11y-group.ts:53]: Yes, thank you @dshovchko @ala-n ! That's the answer I needed!

**Pull request merged**